yojenkins/cli/cli_build.py

```diff
@@ -46,7 +46,10 @@ def info(profile: str, token: str, job: str, number: int, url: str, latest: bool
         url:     The build url to get info on
         latest:  Option to get the latest build
     """
-    if job and not number and not latest:
+    if url is None and job and is_complete_build_url(job):
+        url = job
+        job = None
```

Comment:
Good idea. But, I'm actually trying to have most of the work done in the level below (ie. `yj_obj.build.info(...)`), I'm kind of trying to keep this file as transferable to other `cli_*.py` type of files, if possible.

With that being said, I still don't see anything too wrong here.

Just a minor change though, can you change all these to the following:
```python
url, job = job, None
```

Reply:
I agree! Thank you again! Your contributions are always much appreciated. Let me test a few times on my end here in a bit, and I will release a new version.